src/main/resources/application.yml

```diff
@@ -8,6 +8,10 @@ scheduler:
   notification:
     cron: 0 * * * * *
     shedlock-lock-at-most-for: PT10M
+  suspension:
+    cron: 0 * * * * *
```

Comment:
Should we offset jobs with a couple of minutes to spread the load that is generated?

Reply:
Offsetting is a good idea but that will also the rate these jobs are running.